### PR TITLE
Fix footer content overlapping POA documents list

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
1. Initialise the SDK with Proof of Address (POA) enabled
1. Navigate to the Proof of Address verification document selection screen
1. The footer with "Powered by Onfido" text with the Onfido logo overlaps the documents list

# Solution
- Brought back code for `.scrollableContent` in JSX
- Add flex styling to make the `.scrollableContent` element fill the available space and only scrollable if child content (the list) is more than the height

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [x] Have commented out tests that were failing because of this bug been uncommented?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/DocumentSelector/style.css [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+.listContainer {
+  margin: 8*@unit @large-text-margin 0;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  padding: 0;
-  position: relative;
-  margin: 8*@unit @large-text-margin 0;
-  flex-shrink: 0;
+  justify-content: flex-start;
```
Comment (stefaniacardenas) [inline src/components/DocumentSelector/style.css]: Looks good. Do you think we need to extract this into a class and extend all the containers that need this type layout?

Comment (kopijunkie) [reply src/components/DocumentSelector/style.css]: Probably could if we think there's already a few containers that need it. Maybe as a separate PR?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
as discussed, please make sure the SDK occupies a fixed height beyond the desktop breakpoint i.e. check out https://5-0-0-tag-onfido-sdk-ui-onfido.surge.sh/?poa=true

File: src/components/DocumentSelector/style.css (hunk 1 above)
Comment (kopijunkie) [reply src/components/DocumentSelector/style.css]: Slight change to the code based on See Wah's feedback.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-746-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://746-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**